### DISCUSSION TITLE
Is there a way to adjust the buffered scroll default trigger value on w2ui grid?

### DISCUSSION THREAD

**Discussion by nickbrady68** [vitmalina/w2ui#2419] [Q&A]
Currently using version 2.0.
I have rendered a table which contains a few thousand rows.
When scrolling on devices which are on the slow side, it seems the buffered render causes the table to "jump" unexpectedly.
I was wondering if there is a way to adjust the value at which point to trigger the buffered scroll?
Alternatively is there a way to increase the number of records that are rendered when scrolling into view?

**Comment by vitmalina** [comment]
I think so. See these two links https://w2ui.com/web/docs/2.0/w2grid.vs_start and https://w2ui.com/web/docs/2.0/w2grid.vs_extra.
